### ISSUE TITLE
cryptic fatal error message for nonexistent files since 6.9.0

### ISSUE THREAD

**Issue by kartiksubbarao** [johnkerl/miller#1510]
Before 6.9.0, mlr used to handle nonexistent files with a clear error message:
```
% ./miller-6.8.0-linux-amd64/mlr -t cat nonexistent
mlr: open nonexistent: no such file or directory.
```
However, since version 6.9.0, mlr crashes with a cryptic fatal error in these situations:

```
% ./miller-6.11.0-linux-amd64/mlr -t cat nonexistent
fatal error: all goroutines are asleep - deadlock!

goroutine 1 [select]:
github.com/johnkerl/miller/pkg/stream.Stream({0xc0000ada10, 0x1, 0x1}, 0xc000204540, {0xc0000b8f20, 0x2, 0x2}, {0x9f8250?, 0xc000012018}, 0x1)
	github.com/johnkerl/miller/pkg/stream/stream.go:95 +0x6b7
github.com/johnkerl/miller/pkg/entrypoint.processToStdout(...)
	github.com/johnkerl/miller/pkg/entrypoint/entrypoint.go:81
github.com/johnkerl/miller/pkg/entrypoint.Main()
	github.com/johnkerl/miller/pkg/entrypoint/entrypoint.go:60 +0x18b
main.main()
	github.com/johnkerl/miller/cmd/mlr/main.go:78 +0x5fe

goroutine 9 [chan receive]:
github.com/johnkerl/miller/pkg/transformers.runSingleTransformer({0x9f5bc0, 0xc0000c4be0}, 0x45?, 0x8f92b5?, 0x7?, 0x0?, 0x0?, 0x0?)
	github.com/johnkerl/miller/pkg/transformers/aaa_chain_transformer.go:211 +0x5f
created by github.com/johnkerl/miller/pkg/transformers.ChainTransformer
	github.com/johnkerl/miller/pkg/transformers/aaa_chain_transformer.go:187 +0x2ea

goroutine 8 [chan receive]:
github.com/johnkerl/miller/pkg/output.ChannelWriter(0x0?, {0x9f5ac0, 0xc0001ec570}, 0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
	github.com/johnkerl/miller/pkg/output/channel_writer.go:24 +0x65
created by github.com/johnkerl/miller/pkg/stream.Stream
	github.com/johnkerl/miller/pkg/stream/stream.go:89 +0x5e8

goroutine 10 [chan receive]:
github.com/johnkerl/miller/pkg/transformers.runSingleTransformer({0x9f5ce0, 0xc0001ec540}, 0x0?, 0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
	github.com/johnkerl/miller/pkg/transformers/aaa_chain_transformer.go:211 +0x5f
created by github.com/johnkerl/miller/pkg/transformers.ChainTransformer
	github.com/johnkerl/miller/pkg/transformers/aaa_chain_transformer.go:187 +0x2ea
```

**Comment by johnkerl** [comment]
@kartiksubbarao thank you! This was fixed in #1508 and will be delivered in an upcoming 6.12.0 release.

**Comment by kartiksubbarao** [comment]
Awesome, thanks @johnkerl!
